packages/twenty-server/src/engine/workspace-manager/workspace-migration-runner/workspace-migration-runner.service.ts

```diff
@@ -89,7 +89,7 @@ export class WorkspaceMigrationRunnerService {
 
       await queryRunner.commitTransaction();
     } catch (error) {
-      this.logger.error('Error executing migration', error);
+      console.error('Error executing migration', error);
```

Comment:
logger swallow the errors too much which is a problem to troubleshoot in prod

Reply:
Interesting